core/domain/entities/admin/GraphQLData/GraphQLData.php

```diff
@@ -63,15 +63,21 @@ protected function getQueryResponse($field_key, array $where_params = [])
      */
     protected function makeGraphQLRequest($data)
     {
+        $error = '';
         try {
             $response = graphql($data);
             if (! empty($response['data'])) {
                 return $response['data'];
             }
-            return null;
         } catch (Exception $e) {
-            // do something with the errors thrown
-            return null;
+            if (defined('GRAPHQL_DEBUG') && GRAPHQL_DEBUG) {
+                $error = $e->getMessage();
+            }
+        }
+        if (!$error && ! empty($response['errors'])) {
+            $error = print_r($response['errors'], true);
         }
+        error_log($error);
+        return null;
```

Comment:
@tn3rb I think this can be improved a bit because the error log will not mention the trace. So it's better to add the backtrace.

Reply:
Any comment on this?